Index: Doc/Class.puml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@startuml\n'https://plantuml.com/class-diagram\n!pragma layout smetana\n\nclass Main{\n    + static void main(String[] args)\n}\nclass Datamon{\n    + void runGame()\n}\n\npackage \"util\"{\n    class TextUI{\n        + void displayMessage(String msg)\n        + void getInput (String msg)\n    }\n    class DBConnector{\n        + void loadUsers\n        + void createUser(String username, String password)\n        + void login(String username, String password)\n\n    }\n}\npackage \"model\" {\n    class user{\n\n    }\n    interface Mon{\n\n    }\n    class ActualMons implements Mon{\n\n    }\n}\n\npackage \"JavaFX\"{\n    class Controllers\n}\n\nMain -- Datamon\nDatamon-- Mon\nDatamon -- TextUI\nDatamon-- DBConnector\nDatamon -- user\nDatamon-- Controllers\n\n\n\n\n\n\n\n\n@enduml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Doc/Class.puml b/Doc/Class.puml
--- a/Doc/Class.puml	(revision c5f0a8f5962b4e5e5df04da59f945cc8ad519f1b)
+++ b/Doc/Class.puml	(date 1702234583364)
@@ -37,11 +37,11 @@
     class Controllers
 }
 
-Main -- Datamon
+Main-- Datamon
 Datamon-- Mon
 Datamon -- TextUI
 Datamon-- DBConnector
-Datamon -- user
+Datamon-- user
 Datamon-- Controllers
 
 
