Index: Doc/Domain.puml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@startuml\n'https://plantuml.com/object-diagram\n!pragma layout smetana\n\nobject Mon\nobject User\nobject Datamon\nobject DBConnector\nobject TextUI\nobject JavaFX\nobject Controllers\nobject ActualMons\n\nDatamon o-- Mon\nDatamon o-- JavaFX\nJavaFX o-- Controllers\nDatamon o-- User\nDatamon o-- TextUI\nDatamon o-- DBConnector\nMon <|-- ActualMons\n\n@enduml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Doc/Domain.puml b/Doc/Domain.puml
--- a/Doc/Domain.puml	
+++ b/Doc/Domain.puml	
@@ -19,4 +19,5 @@
 Datamon o-- DBConnector
 Mon <|-- ActualMons
 
-@enduml
\ No newline at end of file
+
+@enduml
Index: Doc/Class.puml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@startuml\n'https://plantuml.com/class-diagram\n!pragma layout smetana\n\nclass Main{\n    + static void main(String[] args)\n}\nclass Datamon{\n    + void runGame()\n}\n\npackage \"util\"{\n    class TextUI{\n        + void displayMessage(String msg)\n        + void getInput (String msg)\n    }\n    class DBConnector{\n        + void loadUsers\n        + void createUser(String username, String password)\n        + void login(String username, String password)\n\n    }\n}\npackage \"model\" {\n    class user{\n\n    }\n    interface Mon{\n\n    }\n    class ActualMons implements Mon{\n\n    }\n}\n\npackage \"JavaFX\"{\n    class Controllers\n}\n\nMain -- Datamon\nDatamon-- Mon\nDatamon -- TextUI\nDatamon-- DBConnector\nDatamon -- user\nDatamon-- Controllers\n\n\n\n\n\n\n\n\n@enduml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Doc/Class.puml b/Doc/Class.puml
--- a/Doc/Class.puml	
+++ b/Doc/Class.puml	
@@ -37,12 +37,12 @@
     class Controllers
 }
 
-Main -- Datamon
-Datamon-- Mon
-Datamon -- TextUI
-Datamon-- DBConnector
-Datamon -- user
-Datamon-- Controllers
+Main *-- Datamon
+Datamon *-- Mon
+Datamon *-- TextUI
+Datamon *-- DBConnector
+Datamon *-- user
+Datamon *-- Controllers
 
 
 
